### PR TITLE
Add confirmation page for 1995 form

### PR DESCRIPTION
**Pull request by jbalboni**
Related to #4608 

This adds the common code we need to display the confirmation page after submitting a form. It's not complete for the 1995 form because I'm not sure of all the information that should be displayed, but that can be a separate issue.

The changes are:

- Cleaned up submit action to use analytics event name and transformation functions from form config
- Updated review page to move to the confirmation page after a successful submit
- Created 1995 confirmation page.

screen shot 2017-01-20 at 3 33 27 pm

### CODE REVIEW

File: src/js/common/schemaform/actions.js [hunk 1]
```diff
@@ -60,21 +62,17 @@ export function submitForm(formConfig, form) {
...
       if (res.ok) {
         window.dataLayer.push({
-          event: 'edu-submission-successful',
+          event: `${formConfig.trackingPrefix}-submission-successful`,
         });
         return res.json();
       }
       window.dataLayer.push({
-        event: 'edu-submission-failed',
+        event: `${formConfig.trackingPrefx}-submission-failed`,
```
Comment (annekainicUSDS) [inline src/js/common/schemaform/actions.js]: Typo on `trackingPrefix` (missing second `i`)

File: src/js/edu-benefits/1995/config/form.js [hunk 2]
```diff
@@ -4,15 +4,18 @@ import { fullName, ssn, dateRange, date, address, phone } from '../../../common/
...
-import { benefitsLabels } from '../helpers';
+import { benefitsLabels, transformForSubmit } from '../helpers';
 import IntroductionPage from '../components/IntroductionPage';
+import ConfirmationPage from '../containers/ConfirmationPage';
 import ServicePeriodView from '../components/ServicePeriodView';
 
 const formConfig = {
   urlPrefix: '/1995/',
   submitUrl: '/v0/education_benefits_claims/1995',
+  trackingPrefx: 'edu',
```
Comment (annekainicUSDS) [inline src/js/edu-benefits/1995/config/form.js]: Typo

Comment (annekainicUSDS) [inline src/js/edu-benefits/1995/config/form.js]: Also do we want to call this something more specific? Like specifying that these events are for the 1995?

**Review by annekainicUSDS — COMMENTED** [review]
LGTM! Couple of small nits.

File: src/js/edu-benefits/1995/config/form.js (hunk 2 above)
Comment (jbalboni) [inline src/js/edu-benefits/1995/config/form.js]: Yep, that's a good point.

File: src/js/common/schemaform/actions.js (hunk 1 above)
Comment (jbalboni) [inline src/js/common/schemaform/actions.js]: Autocomplete is spreading my typos throughout my code :)

**Pull request merged**